common/utils/RsaKeyGenerator.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const {
+  spawn
+} = require('child_process');
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const uuid = require('uuid');
+
+const fileExists = Promise.promisify(fs.access);
```

Comment:
@jagadish-kb  had a point that we should not mix promises and async in the same file. Here we are using both. @jagadish-kb Can you comment please? 

Let us have a guideline to be followed and agreed on.

Reply:
Are you talking about "promisifying" here? If that's the case, I disagree because the "fs" library functions are callback-based only.
Furthermore, I would disagree in general to this because though this sounds good on paper, we would never be able to bring in the benefits of async/await e.g. in Bosh ssh, this is very useful to have

Reply:
I am saying in general about usage of Promise and async in same file. Let us have a guideline to follow.